proto/one/operations/sample/analyte_result.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "jsonTicksDateTime.proto";
+import "enum_import_status.proto";
+option csharp_namespace ="ONE.Models.CSharp";
+
+message AnalyteResult {
+
+    // AuthTwinRefId
+    google.protobuf.StringValue authTwinRefId = 1;
+    
+    // The Sample Number (letter and numbers) that describe the sample
+    string sampleNumber = 2;
+    
+    // The name of the analyte.  (The subject of the activity)
+    string analyteName = 3;
+
+    // Analysis Result.  May include qualifier
+    string result = 4;
+
+    // Excuse Qualifier:  See:  https://aquaticinformatics.atlassian.net/wiki/spaces/AOS/pages/33395971/Glossary+Excuse+Qualifiers
+    google.protobuf.StringValue excuseQualifier = 5;
+    
+    // Lab Qualifier: See: https://aquaticinformatics.atlassian.net/wiki/spaces/AOS/pages/33392022/Glossary+Reportable+Qualifiers
+    google.protobuf.StringValue labQualifier = 6;
+
+    // Method Detection Limit
+    google.protobuf.DoubleValue methodDetectionLimit = 7;   
+
+    // Reporting Limit
+    google.protobuf.DoubleValue reportingLimit = 8;
+    
+    // ReceivedInLab
+    JsonTicksDateTime receivedInLab = 9;
+    
+    // CollectionDate
+    JsonTicksDateTime collectionDate = 10;
+
+    // Analysis Start
+    JsonTicksDateTime analysisStart = 11;
+    
+    // Analysis End
+    JsonTicksDateTime analysisEnd = 12;
+    
+    // Name of person that performed the analysis
+    google.protobuf.StringValue analyzedBy = 13;
+    
+    // Comment for the analysis
+    google.protobuf.StringValue comment = 14;
+    
+    // Import File Id
+    google.protobuf.StringValue importFileId = 15;
+    
+    // Import Status Enum
+    EnumImportStatus importStatus = 16;
+
+    // The string details of the import error(s).
+    google.protobuf.StringValue errorDetails = 17;
```

Comment:
Should this be a repeated string?

Reply:
Fixed. I was thinking it would be one string like how we sometimes do logging but for this, multiple strings would be cleaner.